src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
+  const dateObj = {};
+  const result = [];
+
+  for (let i = 0; i < dateArr.length; i++) {
+    if (fromFormat[i] === 'YY') {
+      if (dateArr[i] < '30') {
+        dateArr[i] = '20'.concat(dateArr[i]);
+      } else {
+        dateArr[i] = '19'.concat(dateArr[i]);
+      }
+      fromFormat[i] = 'YYYY';
+    }
+
+    Object.assign(dateObj, { [fromFormat[i]]: dateArr[i] });
```

Comment:
Instead of using Object.assign for a single new property, you can use simple assignment. For example: dateObj[fromFormat[i]] = dateArr[i].